src/sinks/sematext/metrics.rs

```diff
@@ -122,6 +122,10 @@ impl SinkConfig for SematextMetricsConfig {
     fn sink_type(&self) -> &'static str {
         "sematext_metrics"
     }
+
+    fn can_acknowledge(&self) -> bool {
+        false
```

Comment:
This one seems to be using `HttpBatchService` and implements our `Service` which I think means it should support finalization correctly?

Reply:
Fixed in 392697054